docs/index.md

```diff
@@ -4,17 +4,17 @@ layout: page
 title: GOV.UK Vocabulary
 ---
 
-Human-readable documentation will follow.
-
-The following JSON Schemas are provided:
+### Schemas
```

Comment:
```suggestion
# GOV.UK One Login vocabulary

## Introduction

This vocabulary covers a number of use cases in the One Login domain, including:

* identity proofing and verification
* verifiable credential issuance, in particular of [identity check credentials](v1/IdentityCheckCredentialClass)
* [identity presentation](v1/CoreIdentityJWTClass)
* OAuth 2.0 [authorization requests](v1/AuthorizationRequestClass) to credential issuing and credential collecting services

In future it may be extended to cover OpenID Connect authentication requests and security events.

## Schemas
```